README.md

```diff
@@ -1 +1,75 @@
-# module-ballerina-io
\ No newline at end of file
+Ballerina IO Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-io/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-io/actions?query=workflow%3ABuild)
+  [![Daily build](https://github.com/ballerina-platform/module-ballerina-io/workflows/Daily%20build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-io/actions?query=workflow%3A%22Daily+build%22)
+  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-io.svg)](https://github.com/ballerina-platform/module-ballerina-io/commits/master)
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+The IO library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+
+For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/byte-io.html).
+
+## Building from the Source
+
+### Setting Up the Prerequisites
+
+* Download and install Java SE Development Kit (JDK) version 8 (from one of the following locations).
+
+   * [Oracle](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html)
+   
+   * [OpenJDK](http://openjdk.java.net/install/index.html)
+   
+        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+
+### Building the Source
+
+Execute the commands below to build from source.
+
+1. To build the library:
+        
+        ./gradlew clean build
+
+2. To run the integration tests:
+
+        ./gradlew clean test
+
+3. To build the module without the tests:
+
+        ./gradlew clean build -x test
+
+4. To run only specific tests:
+
+        ./gradlew clean build -Pgroups=<Comma separated groups/test cases>
+
+   **Tip:** The following groups of test cases are available.<br>
+   Groups | Test Cases
+   ---| ---
+   connection | connection
+   pool | pool
+   transaction | transaction
+   execute | execute-basic <br> execute-params
+   batch-execute | batch-execute 
+   query | query-simple-params<br>query-numeric-params<br>query-complex-params
```

Comment:
Unnecssary

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/3/commits/5fe5749ebbad0844e3e310d0a23f0f004862e7b5